test/verify/check-storage-hidden

```diff
@@ -35,18 +36,16 @@ class TestStorageHidden(StorageCase):
 
         self.login_and_go("/storage")
 
-        m.add_disk("50M", serial="DISK1")
-        m.add_disk("50M", serial="DISK2")
-        b.wait_in_text("#drives", "DISK1")
-        b.wait_in_text("#drives", "DISK2")
+        disk1 = self.add_ram_disk()
+        b.wait_in_text("#drives", disk1)
 
         # Create a volume group with a logical volume with a encrypted
         # filesystem.
 
         self.devices_dropdown('Create LVM2 volume group')
```

Comment:
When creating a new volume group in a non-destructive test, we usually add a cleanup function to remove it in case of test failure, no?

Reply:
Like this:

```
        # just in case the test fails
        self.addCleanup(m.execute, "vgremove --force vgroup0 2>/dev/null || true")
```

Reply:
Thanks, can do. I had assumed that VGs would go away when their underlying storage devices disappear. I'll provoke a failure to check this.

Reply:
> I had assumed that VGs would go away when their underlying storage devices disappear.

Yeah, I guess they do. I am not sure why other tests have the cleanup function.